Cargo.toml

```diff
@@ -7,6 +7,8 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+dotenv = "0.15.0"
```

Comment:
let's not add more libraries just for reading a simple config file

Reply:
ok